### PR TITLE
Eventlog-2 updated with device user add/delete/update

### PR DESCRIPTION
**Pull request by vipinlalcm**
Event log code has been added for device user add/delete/update and some bug fixing in that methods.

### CODE REVIEW

**Comment by stemid** [comment]
There are merge conflicts. Can you make sure you have the latest commit of eventlog-2 before you attempt the merge? I've made updates to eventlog-2 that you might have missed.

**Comment by vipinlalcm** [comment]
conflicts has been fixed.

File: siptrackd_twisted/attribute.py [hunk 1]
```diff
@@ -50,11 +57,11 @@ def xmlrpc_set_value(self, session, oid, value):
                 value = str(value)
             except:
                 raise siptrackdlib.errors.SiptrackError('attribute value doesn\'t match type')
-        attribute._set_value(value, user=session.user)
```
Comment (stemid) [inline siptrackd_twisted/attribute.py:53]: @lalusvipi What was the reasoning behind removing the user argument to _set_value? I'm not saying it's wrong or right, just trying to understand. I assume you were not getting the session.user attributed passed in to xmlrpc_set_value as you were supposed to.

**Review by stemid — APPROVED** [review]
@lalusvipi please check my question on the code in your notifications. But I'll approve this for now so we can move on and get it deployed to staging.

File: siptrackd_twisted/attribute.py (hunk 1 above)
Comment (vipinlalcm) [reply siptrackd_twisted/attribute.py:53]: I was getting attached error while updating any of the attribute value.

image

On further checking the _set_value does not need user to set the attribute values here. Instead it needs only the value of attribute to change.
https://github.com/sii/siptrackd/blob/fe03bc399010806dcff282644072546081509467/siptrackdlib/attribute.py#L120

Comment (stemid) [reply siptrackd_twisted/attribute.py:53]: Looks like you're right, Attribute._set_value doesn't even take a user argument. Good catch I guess, weird that it evaded us for so long.

**Pull request merged**